packages/apsara-ui/src/Notification/Notification.tsx

```diff
@@ -31,52 +31,64 @@ export const useNotification = () => {
 };
 
 export const NotificationProvider = ({ children }: any) => {
-    const [toasts, setToasts] = useState<Notification[]>([]);
-    
-    const showNotification = useCallback((toast: Notification) => {
-        setToasts([...toasts, { ...toast, id: uuid() }]);
-    }, [toasts, setToasts]);
+    const [queue, setQueue] = useState<Notification[]>([]);
 
-    const showSuccess = useCallback((title: string, content?: string) => {
-        setToasts([
-            ...toasts,
-            {
-                title: title,
-                content: content,
-                id: uuid(),
-                icon: <Icon name="checkcircle" color="green" size={32} />,
-            },
-        ]);
-    }, [toasts, setToasts]);
+    const showNotification = useCallback(
+        (toast: Notification) => {
+            setQueue((toasts) => [...toasts, { ...toast, id: uuid() }]);
+        },
+        [setQueue],
+    );
+
+    const showSuccess = useCallback(
+        (title: string, content?: string) => {
+            setQueue((toasts) => [
```

Comment:
let's rename it to `currentNotifications` to be more explicit?

```suggestion
            setQueue((currentNotifications) => [
```

Reply:
renamed it to `prevNotifications` instead